### PR TITLE
jupyter extension - add tests for env var configuration

### PR DESCRIPTION
**Pull request by kevyang**
[INT-1229](https://pachyderm.atlassian.net/browse/INT-1229)

[INT-1229]: https://pachyderm.atlassian.net/browse/INT-1229?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: jupyter-extension/jupyterlab_pachyderm/handlers.py [hunk 1]
```diff
@@ -664,11 +665,11 @@ def setup_handlers(web_app, config_file: Path):
             f"Created Pachyderm client for {client.address} from local config"
         )
     except FileNotFoundError:
-        if PACHD_ADDRESS:
```
Comment (bbonenfant) [inline jupyter-extension/jupyterlab_pachyderm/handlers.py:667]: unrelated to this PR, but just clarifying. We want to use pachd_address/dex_token passed through env vars only if no config file is present? And then it doesn't appear that this is written to a config file - should it be?

File: jupyter-extension/jupyterlab_pachyderm/tests/conftest.py [hunk 2]
```diff
@@ -30,7 +30,18 @@ def pach_config(request, tmp_path) -> Path:
 
 
 @pytest.fixture
-def app(pach_config) -> tornado.web.Application:
+def pach_config_env_var(request) -> str:
```
Comment (bbonenfant) [inline jupyter-extension/jupyterlab_pachyderm/tests/conftest.py:33]: Bit of a nit, but this name seems misleading to me, and the marker name seems a little vague. Could they be something like `pachd_address_env_var`?

**Review by bbonenfant — APPROVED** [review]
Looks good!

File: jupyter-extension/jupyterlab_pachyderm/handlers.py (hunk 1 above)
Comment (kevyang) [reply jupyter-extension/jupyterlab_pachyderm/handlers.py:667]: that is correct. good call on this not writing to config file, i think it makes sense to write it. i have added that logic and added testing for it.

**Pull request merged**